cypress/integration/form.js

```diff
@@ -78,4 +78,20 @@ context('Form', () => {
 		cy.get('@row2').click();
 		cy.get('@email_input2').should('not.have.class', 'invalid');
 	});
+
+	it('Shows version conflict warning', { scrollBehavior: false }, () => {
+		cy.visit('/app/todo');
+
+		cy.insert_doc("ToDo", {"description": "old"}).then(doc => {
+			cy.visit(`/app/todo/${doc.name}`);
+			// make form dirty
+			cy.fill_field("status", "Cancelled", "Select");
```

Comment:
idk why we HAVE to specify these fieldtypes ourselves, command should somehow figure this out OR there should be different commands for different types 👀